src/NewTaskView.cpp

```diff
@@ -45,11 +45,12 @@ NewTaskView::MessageReceived(BMessage* message)
 	case BUTTON_NEW_TASK_OK:
 	{
 		BMessage* Temp;
-		if (m_TextControl->Text() == "")
+		if (m_TextControl->Text()[0] == '\0')
```

Comment:
Why the change from `if (m_TextControl->Text() == "")`?
(Another possibilty would be `if (m_TextControl->TextLength() == 0)` btw.)

Reply:
@humdingerb The "==" operator doesn't seem to work correctly with char*, so if I leave it with `if (m_TextControl->Text() == "")`, a new task with no name is added when I leave the textbox blank and press OK.

Reply:
@humdingerb I tried `if (m_TextControl->TextLength() == 0)`, but it does the same thing as `if (m_TextControl->Text() == "")`. I think I'll leave it as it is 😄